controllers/cinder_controller.go

```diff
@@ -911,6 +931,22 @@ func (r *CinderReconciler) schedulerDeploymentCreateOrUpdate(ctx context.Context
 }
 
 func (r *CinderReconciler) backupDeploymentCreateOrUpdate(ctx context.Context, instance *cinderv1beta1.Cinder) (*cinderv1beta1.CinderBackup, controllerutil.OperationResult, error) {
+
+	bckTemplate := instance.Spec.CinderBackup
```

Comment:
At first the "bck" prefix confused me, mainly because cinder tends to use "bak" to abbreviate the backup service. I'd request you switch to "bak" except, as I stated in an earlier comment, I think "svcTemplate" could be used in all 4 of the xxxDeploymentCreateOrUpdate functions.

Reply:
+1 on replacing bck w/ bak to stay consistent.

Reply:
Done, feel free to Resolve the conversation if the change it's ok.